### PR TITLE
Design my account page

### PR DESCRIPTION
**Pull request by YaronMiro**
#89

@amitaibu PR is ready!

* Desktop:
selection_189

* Mobile:
selection_187

### CODE REVIEW

**Comment by amitaibu** [comment]
Needs a re-roll

**Comment by YaronMiro** [comment]
**Updated status:**
@amitaibu 
PR is ready.

* Desktop:
selection_021


* Mobile:
selection_023

**Comment by amitaibu** [comment]
Merged, thanks.

File: src/elm/User.elm [hunk 1]
```diff
@@ -152,16 +152,28 @@ view address model =
...
+          , text " My account"
+          ]
       in
-        div [class "container"]
-          [ div [] [ text "Welcome ", italicName ]
-          , div [] [ text "Your companies are:"]
-          , ul  [] (List.map viewCompanies model.companies)
+        div
+          [
+          id "my-account"
```
Comment (amitaibu) [inline src/elm/User.elm]: this should be

```elm
div
  [ id ...
```

**Pull request closed without merging**